remote-api/src/main/java/org/alfresco/rest/framework/resource/content/BinaryProperty.java

```diff
@@ -73,6 +73,7 @@ public BinaryProperty(ContentReader reader)
         this.encoding = reader.getEncoding();
         this.length = reader.getSize();
         this.locale = reader.getLocale();
+        this.storageClasses = Collections.emptySet();

```

Comment:
Is this ok to an empty set? How do we distinguish when we set the value from when we do not?

Reply:
Updated.